### ISSUE TITLE
deployment: default files location not created automatically

### ISSUE THREAD

**Issue by nharraud** [EUDAT-B2SHARE/b2share#1419]
**Problem:**
When the docker deployment is created with `LOAD_DEMO_COMMUNITIES_AND_RECORDS=0` or unset, there is no file location created. The result is that it is not possible to create buckets, which means that it is not possible to create draft records.

**Temporary solution:**
Once the deployment is started, start a shell on the b2share docker container.
```sh
$ docker exec -it dockerize_b2share_1 /bin/bash
```
Then create the location
```
b2share files add-location 'local' file:///usr/var/b2share-instance/files --default
```

**Comment by emanueldima** [comment]
@nharraud Solved? or do you want to have it fixed in code?

**Comment by nharraud** [comment]
Yes this is fixed for 2.0.1 but I don't think #1420 is fixed as people using v2.0.0 will still have the issue.
